crates/bazed/src/main.rs

```diff
@@ -1,3 +1,10 @@
-fn main() {
-    panic!("hello world");
```

Comment:
Sad to see it go.

Reply:
Yes, maybe it should be hidden behind a feature flag?